file-id/src/lib.rs

```diff
@@ -85,21 +85,21 @@ pub enum FileId {
 
 impl FileId {
     pub fn new_inode(device_id: u64, inode_number: u64) -> Self {
-        FileId::Inode {
+        Self::Inode {
```

Comment:
I think replacing the Name with Self is only reducing readability and doesn't really remove any duplication (FileId -> Self) when you use also `use FileId::*;` and drop the prefix completely. Either way I think it's fine as it is.